src/views/MarketsDropdown.tsx

```
@@ -47,335 +47,338 @@ import { MarketFilter } from './MarketFilter';
-      };
-
-      window.addEventListener('keydown', handleKeyDown);
 
-      return () => {
-        window.removeEventListener('keydown', handleKeyDown);
-      };
-    }, []);
+        return null;
+    }, [filter, stringGetter]);
+
+    const [hasSeenElectionBannerTrumpWin, setHasSeenElectionBannerTrupmWin] = useLocalStorage({
```

ESLint

[error] Delete `··`

(prettier/prettier)